photon-client/src/main/scala/com/linkedin/photon/ml/estimators/GameEstimator.scala

```diff
@@ -55,7 +55,7 @@ class GameEstimator(val sc: SparkContext, val params: GameParams, implicit val l
   import GameEstimator._
 
   // 2 types that makes the code more readable
-  // TODO: Those look like they should be in file Types?
+  // TODO: Should they be in file Types?
```

Comment:
Nope - which is why I put them there. Also, if we put all types under the sun in `Types`, the code can become less readable in some places :-/